docs/topics/proc_deploying-the-crud-booster-using-the-oc-cli-client.adoc

```diff
@@ -31,6 +31,8 @@ $ oc new-project {project-name}
 . Navigate to the root directory of your booster.
 
 . Deploy the PostgreSQL database to OpenShift.
+Ensure that you use the following login credentials when creating the database application.
```

Comment:
Personally, I've never been a fan of the word "credentials", and it's not just credentials here, it's also the database name. So I'd replace "login credentials" with "username, password and database name".

Reply:
@Ladicek I agree that 'credentials' might not be the most accurate term here. I'll make the change here.